catalog/ui/search-ui/search-ui-app/src/main/resources/features.xml

```diff
@@ -70,11 +70,15 @@
         </configfile>
     </feature>
 
+    <feature name="simple-search-ui" install="auto" version="${project.version}">
```

Comment:
Is there no dependency between `search-ui` and `simple-search-ui`? Previously, this feature didn't exist but its bundle was installed as part of `search-ui`. Now, they are both started automatically but there's no order guarantees.